app/src/main/java/com/example/farmeraid/data/CharityRepository.kt

```diff
@@ -13,6 +13,10 @@ class CharityRepository (
     private val userRepository: UserRepository
 ){
     private var db: FirebaseFirestore = FirebaseFirestore.getInstance()
+    data class ProduceFridge(
+        val produceName: String,
+        val produceDonateAmount: Int,
+    )
```

Comment:
Create a CharityModel and put this in there, since we are going to be using this class in other files as well, it'll be a cleaner import.